### ISSUE TITLE
Failed to run `yarn test:prod`

### ISSUE THREAD

**Issue by gurrpi** [quadratic-funding/mpc-phase2-suite#349]
## Expected

Pass all tests

## Actual

Failed

### Notable errors

1. 
```
FAIL packages/actions/test/unit/verification.test.ts (5.924 s)
  ● Verification utilities › generateGROTH16Proof › should generate a GROTH16 proof

    FirebaseError: Firebase: Error (auth/operation-not-allowed).

      at createErrorInternal (node_modules/@firebase/auth/src/core/util/assert.ts:137:7)
      at _fail (node_modules/@firebase/auth/src/core/util/assert.ts:65:28)
      at node_modules/@firebase/auth/src/api/index.ts:177:9
      at step (node_modules/tslib/tslib.js:144:27)
      at Object.next (node_modules/tslib/tslib.js:125:57)
      at fulfilled (node_modules/tslib/tslib.js:115:62)
```

2. 
```
  ● Contribute › getCeremonyCircuits › should return the circuits for the specified ceremony

    7 PERMISSION_DENIED: Cloud Firestore API has not been used in project 729940941854 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/firestore.googleapis.com/overview?project=729940941854 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.

      130 |         .collection(commonTerms.collections.ceremonies.name)
      131 |         .doc(ceremonyData.uid)
    > 132 |         .set({
          |          ^
      133 |             ...ceremonyData.data
      134 |         })
      135 |
```

3.
```
 ● Test suite failed to run

    5 NOT_FOUND: The database (default) does not exist for project p0tion-dev-branch Please visit https://console.cloud.google.com/datastore/setup?project=p0tion-dev-branch to add a Cloud Datastore or Cloud Firestore database.

      153 |     circuitId: string
      154 | ) => {
    > 155 |     await adminFirestore.collection(getCircuitsCollectionPath(ceremonyId)).doc(circuitId).delete()
          |                                                                                                 ^
      156 |     await adminFirestore.collection(commonTerms.collections.ceremonies.name).doc(ceremonyId).delete()
      157 | }
      158 |
```

## Env

p0tion version: d1cd430177db0a65d644d95fda40dc2192ba1680
Firebase env
 * fresh Firebase project
 * upgraded to Blaze plan
 * Auth set only GitHub

## Log

See https://gist.github.com/gurrpi/59bf897eeaa1e1bc19556108cca1fbcb

**Comment by gurrpi** [comment]
There are another failures with the environment version with f1335caf60ccf9bb38a829992e01185d134db93c:

https://github.com/quadratic-funding/mpc-phase2-suite/actions/runs/4478138590/jobs/7872292098

**Comment by baumstern** [comment]
Fixed in #351 #363 #364
